core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -1147,6 +1151,28 @@ public String stripPrefixIfPresent(String path) {
     T apply() throws IOException;
   }
 
+  /**
+   * Filter exception that need to be retried.
```

Comment:
alluxio convention is to use the third person singular

Reply:
fixed